internal/cli/atlas/privateendpoints/describe.go

```diff
@@ -25,14 +25,15 @@ import (
 )
 
 var describeTemplate = `ID	ENDPOINT SERVICE	STATUS	ERROR
-{{.ID}}	{{.EndpointServiceName}}	{{.Status}}	{{.ErrorMessage}}
+{{.ID}}{{if .EndpointServiceName}}	{{.EndpointServiceName}}{{else}}	PrivateLinkServiceName{{end}}	{{.Status}}	{{.ErrorMessage}}
```

Comment:
[nit] you can move the tab outside of the if and have it just once
```suggestion
{{.ID}}	{{if .EndpointServiceName}}{{.EndpointServiceName}}{{else}}PrivateLinkServiceName{{end}}	{{.Status}}	{{.ErrorMessage}}
```